tidb-cloud/migrate-from-mysql-using-aws-dms.md

```diff
@@ -0,0 +1,181 @@
+
+11. Click **Create endpoint**.
+
+    ![Click Create endpoint](/media/tidb-cloud/aws-dms-tidb-cloud/aws-dms-to-tidb-cloud-target-endpoint3.png)
+
+## Step 4. Create a database migration task
+
+1. In the AWS DMS console, go to the [Data migration tasks](https://us-west-2.console.aws.amazon.com/dms/v2/home?region=us-west-2#tasks) page. Switch to your region. Then click **Create task** in the upper right corner of the window.
+
+    ![Create task](/media/tidb-cloud/aws-dms-tidb-cloud/aws-dms-to-tidb-cloud-create-task.png)
+
+2. Configure the following information:
+    - **Task identifier**: fill in a name for the task. It is recommended to use a name that is easy to remember.
+    - **Descriptive Amazon Resource Name (ARN) - optional**: create a friendly name for the default DMS ARN.
+    - **Replication instance**: select the AWS DMS instance that you just created.
+    - **Source database endpoint**: select the source database endpoint that you just created.
+    - **Target database endpoint**: select the target database endpoint that you just created.
+    - **Migration type**: select a migration type as needed. In this example, select **Migrate existing data and replicate ongoing changes**.
+
+    ![Task configurations](/media/tidb-cloud/aws-dms-tidb-cloud/aws-dms-to-tidb-cloud-task-config.png)
+
+3. Configure the following information:
+    - **Editing mode**: select **Wizard**.
+    - **Target table preparation mode**: select **Do nothing** or other options as needed. In this example, select **Do nothing**.
+    - **Include LOB columns in replication**: select **Limited LOB mode**.
+    - **Maximum LOB size in (KB)**: use the default value **32**.
+    - **Turn on validation**: select it according to your needs.
+    - **Task logs**: select **Turn on CloudWatch logs** for troubleshooting in future.
+
+    ![Task settings](/media/tidb-cloud/aws-dms-tidb-cloud/aws-dms-to-tidb-cloud-task-settings.png)
+
+4. In the **Table mappings** section, specify the database to be migrated. 
+
+    The schema name is the database name in the Amazon RDS instance. The default value of the source name is "%", which means that all databases in the Amazon RDS will be migrated to TiDB. It will cause the system databases such as `mysql` and `sys` in Amazon RDS to be migrated to TiDB, and result in task failure. Therefore, it is recommended to fill in the specific database name, or filter out all system databases. For example, according to the settings in the following screenshot, only the database named `franktest` and all the tables in that database will be migrated.
+
+5. Click **Create task** in the lower-right corner.
+
+    ![Table mappings](/media/tidb-cloud/aws-dms-tidb-cloud/aws-dms-to-tidb-cloud-table-mappings.png)
```

Comment:
```suggestion
    The schema name is the database name in the Amazon RDS instance. The default value of the source name is "%", which means that all databases in the Amazon RDS will be migrated to TiDB. It will cause the system databases such as `mysql` and `sys` in Amazon RDS to be migrated to TiDB, and result in task failure. Therefore, it is recommended to fill in the specific database name, or filter out all system databases. For example, according to the settings in the following screenshot, only the database named `franktest` and all the tables in that database will be migrated.

        ![Table mappings](/media/tidb-cloud/aws-dms-tidb-cloud/aws-dms-to-tidb-cloud-table-mappings.png)

5. Click **Create task** in the lower-right corner.

```